mudlet-mapper.xml

```diff
@@ -11444,6 +11453,8 @@ function mmp.downloadcrowdmap(newversion)
     downloadFile(mmp.crowdmapfile, "http://ire-mudlet-mapping.github.io/AchaeaCrowdmap/Map/map")
   elseif mmp.game == "starmourn" then
     downloadFile(mmp.crowdmapfile, "http://ire-mudlet-mapping.github.io/StarmournCrowdmap/Map/map")
+  elseif mmp.game == "lusternia" then
```

Comment:
Unrelated, but perhaps we should https for everything now?

Reply:
Github redirects to HTTPS anyways, but in the long run, we probably should.

Reply:
Yeah, more for comfort of anyone reviewing